sidebars/documentation.js

```diff
@@ -313,6 +313,33 @@ module.exports = [
         label: 'TVM instructions',
         href: '/v3/documentation/tvm/instructions',
       },
+      {
+        type: 'category',
+        label: 'TVM Continuations',
+        items: [
+          {
+            type: 'category',
+            label: 'Basics',
+            items: [
+              {
+                type: 'doc',
+                label: 'Basics: IF, c0, cc, save list',
```

Comment:
```suggestion
                label: 'IF, c0, cc, save list',
```

Perhaps, basics do not have to repeat that they're basics :)